### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Bleu-Yves-Sopoude**
In this feature we:

- Added missing JSX elements to the project
- Made the final design matching zeplin's design
- Used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,18 +1,49 @@
...
+    <div className="bookCard row">
+      <div className="details">
+        <div className="bookDetails column">
+          <span className="Text-Style-9 category">Book</span>
+          <span className="Text-Style-5">{title}</span>
+          <span className="Text-Style-8">{author}</span>
+        </div>
+        <div className="buttons">
+          <button type="button" className="Text-Style-8">Comments</button>
+          <button type="button" id={id} onClick={() => dispatch(removeBook(id))} className="Text-Style-8">
```
Comment (FrederickMih) [inline src/components/Book.js:21]: - [ ] Let's kindly make sure that all the items found in the original design should be included in this card. Remember our design should be exactly the same as the one provided in Zeplin's design.
Screenshot from 2023-08-11 13-32-26

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,153 @@
...
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: #fff;
+}
+
+.buttons button:nth-child(-n + 2) {
+  border-right: solid 1px #e8e8e8;
+  padding-right: 0.938rem;
+}
+
```
Comment (FrederickMih) [inline src/App.css]: - [ ] Please let's kindly once more make sure our styling is EXACTLY the same as the one provided on Zeplin. Adjust the logo and user icon to be in line with the card below. Kindly include the missing items missing in the card as mentioned above. I have added some annotations to your design. Kindly compare it with the original design and fix the minor changes.

| Zeplin | Your design |
| --- | --- |
| Bookstore CMS | Screenshot from 2023-08-11 12-38-09 |

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: :raising_hand: @Bleu-Yves-Sopoude , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!





## Highlights :green_circle: 

- PR is professional :+1: 
- Good CSS practices :+1: 


## Required Changes ♻️
**N/A**
_Check the comments under the review._

## Optional suggestions
**N/A**
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply src/components/Book.js:21]: Fixed.

File: src/App.css (hunk 2 above)
Comment (Abenezer-Tilahun) [reply src/App.css]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Bleu-Yves-Sopoude 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**